sections/errorhandling/asyncerrorhandling.md

```diff
@@ -54,6 +98,31 @@ getData(someParameter, function(err, result) {
     }
 });
 ```
+</details>
+
+<details>
+<summary><strong>Typescript</strong></summary>
+
+```typescript
+getData(someParameter, function(err: Error | null, resultA: ResultA) {
+  if(err !== null) {
+    // do something like calling the given callback function and pass the error
+    getMoreData(resultA, function(err: Error | null, resultB: ResultB) {
+      if(err !== null) {
+        // do something like calling the given callback function and pass the error
+        getMoreData(resultB, function(resultC: ResultC) {
+          getMoreData(resultC, function(err: Error | null, d: ResultD) {
+            if(err !== null) {
+              // you get the idea? 

```

Comment:
Just a question about the trailing symbol, what should be displayed here? On my device, I can only see a box 😄

Reply:
Same thing here, I've got an arrow when editing with my IDE but not in my browser. It was in the js example, just copy-pasted the thing. Should we remove it?